src/network/networkd-nexthop.c

```diff
@@ -133,6 +146,16 @@ DEFINE_HASH_OPS_WITH_KEY_DESTRUCTOR(
                 nexthop_compare_func,
                 nexthop_free);
 
+static void nexthop_copy(NextHop *dest, const NextHop *src) {
```

Comment:
this helper doesn't copy all the struct members, so maybe name it something like "nexthop_copy_network_data" to be more specific?

Reply:
Instead of rename the function, I added a brief comment.